src/components/PrintForm/index.tsx

```diff
@@ -225,7 +226,9 @@ export const PrintForm: React.FC<PrintFormProps> = ({
       // Use locale print app if available.
       // Implies that a print app with the language code exists.
       const apps = pManager.getPrintApps();
-
+      if (printApp){
+        await pManager.setPrintApp(printApp);
+      }
       if (apps && currentLanguageCode && apps.includes(currentLanguageCode)) {
         await pManager.setPrintApp(currentLanguageCode);
```

Comment:
This would potentially override the custom printApp. Not sure if this intended?

Reply:
This issue still seems to exist. Do you see any chance to fix it?

Reply:
How do I fix that?

> This issue still seems to exist. Do you see any chance to fix it?

Reply:
The issue is as follows:

- the mapfish print servlet [may serve different print apps](https://mapfish.github.io/mapfish-print-doc/api.html#apps).
- line 232 checks if there is any print app available that contains the current language code (e.g. `de`).
- if so, this print app will be used.

Actually this will override the print app set by the configuration (lines 229 - 231). We should check if this behaviour could be optimized.

Reply:
Ok, thanks for the clarification. I have made the changes